include/revng/ABI/Definition.h

```diff
@@ -0,0 +1,99 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include <vector>
+
+#include "revng/ABI/TypeSpecific.h"
+#include "revng/ADT/SortedVector.h"
+#include "revng/Model/ABI.h"
+#include "revng/Model/Register.h"
+
+/* TUPLE-TREE-YAML
+
+name: Definition
+type: struct
+fields:
+  - name: ABI
+    type: model::ABI::Values
+
+  - name: ArgumentsArePositionBased
+    type: bool
```

Comment:
We had a good deal of docs for this stuff, where did it go?

Reply:
Whoops, I lost them 😅. They'll be back shortly.

Reply:
They are back :tada:: https://github.com/revng/revng/pull/232/commits/1c7a68657fc10dee0d6206b8b3ca13cb350aea08.

But also, to raise a concern, the yaml now feels unreadable. We should probably consider a less intrusive way to introduce docs for the struct members at some point.